app/lib/theme/CustomFields.tsx

```diff
@@ -0,0 +1,212 @@
+import React from "react";
+import { FieldProps } from "@rjsf/core";
+import NumberFormat from "react-number-format";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faLongArrowAltRight } from "@fortawesome/free-solid-svg-icons";
+
+const showStringDiff = (id: string, oldData: string, newData: string) => (
+  <>
+    <span id={id && `${id}-diffOld`} className="diffOld">
+      {oldData}
+    </span>
+    &nbsp;
+    <FontAwesomeIcon size="lg" color="black" icon={faLongArrowAltRight} />
+    &nbsp;
```

Comment:
I think we're not supposed to use `&nbsp` as part of layout instructions, we should use a style with margins on the `<FontAwesomeIcon ... />` instead